Header header
uint32 mapSize
float64 mapRange
uint32 viewingAngle